Tasks/JenkinsQueueJobV2/jenkinsqueuejobtask.ts

```diff
@@ -119,12 +122,18 @@ async function doWork() {
         const taskOptions: TaskOptions = new TaskOptions();
         const jobQueue: JobQueue = new JobQueue(taskOptions);
         const queueUri = await util.pollSubmitJob(taskOptions);
-        console.log(tl.loc('JenkinsJobQueued'));
-        const rootJob = await util.pollCreateRootJob(queueUri, jobQueue, taskOptions);
-        //start the job queue
-        jobQueue.Start();
-        //store the job name in the output variable
-        tl.setVariable('JENKINS_JOB_ID', rootJob.ExecutableNumber.toString());
+
+        if (queueUri == '302') {
+            const jobUrl = util.addUrlSegment(taskOptions.serverEndpointUrl, util.convertJobName(taskOptions.jobName));
+            console.log('Check Jenkins job for new started builds -', jobUrl);
```

Comment:
```suggestion
            console.log('Check Jenkins job for new started builds - ', jobUrl);
```

Reply:
This will add extra whitespace to line on logs. Currently result of the console log is displayed as "Check Jenkins job for new started builds - http://13.40.224.199:8080/job/TempJob1".

Reply:
See it know, thanks! console.log already joins arguments with space